packages/medusa/src/loaders/config.ts

```diff
@@ -34,29 +67,7 @@ export default (rootDirectory: string): ConfigModule => {
     )
   }
 
-  const jwt_secret =
-    configModule?.projectConfig?.jwt_secret ?? process.env.JWT_SECRET
-  if (!jwt_secret) {
-    errorHandler(
-      `[medusa-config] ⚠️ jwt_secret not found.${
-        isProduction
-          ? ""
-          : " fallback to either cookie_secret or default 'supersecret'."
-      }`
-    )
-  }
-
-  const cookie_secret =
-    configModule?.projectConfig?.cookie_secret ?? process.env.COOKIE_SECRET
-  if (!cookie_secret) {
-    errorHandler(
-      `[medusa-config] ⚠️ cookie_secret not found.${
-        isProduction
-          ? ""
-          : " fallback to either cookie_secret or default 'supersecret'."
-      }`
-    )
-  }
+  configModule.projectConfig.auth = buildAuthConfig(configModule.projectConfig)
```

Comment:
I suggest we do this for the entire `projectConfig` somewhere higher in the stack (something like `normalizeConfig` in the `loaders/index.ts` maybe), so you never need to do
```
const somevalue = projectConfig.someExpectedValue ?? "default-value
```
throughout the codebase, and all the defaults are set on load